### ISSUE TITLE
Gradle

### ISSUE THREAD

**Issue by mimminito** [romainpiel/Michelangelo#3]
Is this available in Maven Central yet?

**Comment by romainpiel** [comment]
Nope, I was waiting for some more interest to make the effort ;) Will do it by the end of the week.

**Comment by mimminito** [comment]
No worries, would just make it easier using the library :)

**Comment by intrications** [comment]
Is this on Maven Central now? The latest commit suggests it is.

**Comment by romainpiel** [comment]
Not yet, I can't find time to do it. If anyone is comfortable doing it, go on I've never done that before.

**Comment by vpratfr** [comment]
Just did that for a couple of libraries of mine. I would love to get michelangelo from Gradle. I can guide you if you want to.
